File: lab/lineages/mutation-79__rime/fixtures/src/leftover.js
export function openFile(path, err) {
  throw new Error("open " + path + ": " + err);
}

export function done(body) {
  return body + "\nDone.";
}

export function stamp(ts, body) {
  return ts + " ERROR " + body;
}

export function spawn(cmd) {
  return "failed to spawn `" + cmd + "`";
}

export function fence(response) {
  return "```json\n" + response() + "\n```";
}

export function cleanedWrite(cleaned) {
  return cleaned + "\n";
}

export function failed(op, path, err) {
  return "failed " + op + " on " + path + " with " + err;
}

File: lab/lineages/mutation-79__rime/fixtures/src/leftover.go
package leftover

import "os"

func writeCleaned(path, cleaned string) error {
	return os.WriteFile(path, []byte(cleaned+"\n"), 0o644)
}

func stamp(ts, body string) string {
	return ts + " ERROR " + body
}

func spawn(cmd string) string {
	return "failed to spawn `" + cmd + "`"
}

func done(body string) string {
	return body + "\nDone."
}
